projects/plugins/jetpack/extensions/store/membership-products/resolvers.js

```diff
@@ -71,7 +75,44 @@ const mapAPIResponseToMembershipProductsStoreData = ( response, registry, dispat
 
 const fetchSubscriberCounts = async () => {
 	const response = await apiFetch( {
-		path: '/wpcom/v2/subscribers/counts',
+		path: addQueryArgs( '/wpcom/v2/subscribers/counts', {
+			subscription_status: 'email_subscriber',
+			subscriber_status: 'active_subscriber',
+		} ),
+	} );
+
+	if ( ! response || typeof response !== 'object' ) {
+		throw new Error( 'Unexpected API response' );
+	}
+
+	/**
+	 * WP_Error returns a list of errors with custom names:
+	 * `errors: { foo: [ 'message' ], bar: [ 'message' ] }`
+	 * Since we don't know their names, to get the message, we transform the object
+	 * into an array, and just pick the first message of the first error.
+	 *
+	 * @see https://developer.wordpress.org/reference/classes/wp_error/
+	 */
+	const wpError = response?.errors && Object.values( response.errors )?.[ 0 ]?.[ 0 ];
+	if ( wpError ) {
+		throw new Error( wpError );
+	}
+
+	return response;
+};
+
+const fetchTotalEmailsSentCount = async () => {
+	const { blog_id: siteId } = getSiteData()?.wpcom ?? {};
+	const postId = window.wp.data.select( 'core/editor' ).getCurrentPostId();
+
+	if ( ! siteId || ! postId ) {
+		return;
+	}
+
+	const response = await apiFetch( {
+		path: isSimpleSite()
+			? `/rest/v1.1/sites/${ siteId }/stats/opens/emails/${ postId }/rate`
+			: `/jetpack/v4/stats-app/sites/${ siteId }/stats/opens/emails/${ postId }`,
```

Comment:
I noticed that we need 2 different endpoints to get this data. Let me know if there is a better way to do this.

Reply:
I was curious what the Newsletter debug tool uses and found this lib: fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Syvo%2Srznvy%2Qfgngf%2Srznvy%2Qfhzznel.cuc%3Se%3Q0rq77969-og

Here's where it's used: fbhepr%2Skers%2Szp%2Sarjfyrggref%2Shgvyf.cuc%3Se%3Q501s5942%26zb%3Q2628%26sv%3Q117%23117-og

Do the endpoints use the same lib?

Reply:
In Jetpack we have following stats screen. I noticed that Jetpack is calling different API's for dotcom and self hosted sites so I did the same.

Currenlty I am looking into the code about how Jetpack is making this decision.

![image](https://github.com/user-attachments/assets/b4d9f634-85ba-4043-9a54-c79f27aa90c9)

Reply:
I also find out that we are extensively hard coding `/rest/v1.1/` and `/jetpack/v4/` in codebase so hardcoding these here also seems fine.